build/gen-make-frags/gen-make-frag.sh

```diff
@@ -86,6 +86,9 @@ print_usage()
 	echo "                 root_dir."
 	echo "   -h          hide"
 	echo "                 Hide the makefile fragments by prepending filenames with '.'."
+	echo "   -i LIST     ignore"
+	echo "                 Augment the list of directory names contained in ign_list"
+	echo "                 with the directory names in LIST."
```

Comment:
Like `configure.sh`, here-docs can improve readability and maintainability.
```
cat <<EOF
Enter
Text
Here
EOF
```

Reply:
I agree. :)